flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  description = "booker-flake";
+
+  inputs = { 
+    nixpkgs.url = "nixpkgs/nixos-unstable";
```

Comment:
why did we pick nix `unstable`

Reply:
this is to make sure we have the newest version yarn that is in nixpkgs